packages/augur-ui/src/modules/types.ts

```diff
@@ -125,6 +308,7 @@ export interface LoginAccount {
   dai: string;
 }
 
+type ButtonActionType = (el: React.Component) => void;
```

Comment:
@JohnDanz look at this button action type. What do you think?

Reply:
Isn't the passed object a Click Event, not an element? (if so, that should be an `e` not `el`)